shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -242,17 +174,8 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
     return NO;
   }
 
-  const FlutterRendererConfig rendererConfig = {
-      .type = kOpenGL,
-      .open_gl.struct_size = sizeof(FlutterOpenGLRendererConfig),
-      .open_gl.make_current = (BoolCallback)OnMakeCurrent,
-      .open_gl.clear_current = (BoolCallback)OnClearCurrent,
-      .open_gl.present = (BoolCallback)OnPresent,
-      .open_gl.fbo_with_frame_info_callback = (UIntFrameInfoCallback)OnFBO,
-      .open_gl.fbo_reset_after_present = true,
-      .open_gl.make_resource_current = (BoolCallback)OnMakeResourceCurrent,
-      .open_gl.gl_external_texture_frame_callback = (TextureFrameCallback)OnAcquireExternalTexture,
-  };
+  [_openGLRenderer attachToFlutterView:_viewController.flutterView];
```

Comment:
There is no guarantee that there will be a view here. How are you handling later changes to the view controller given that you removed `setViewController:`?

Reply:
Good point, will account for it.